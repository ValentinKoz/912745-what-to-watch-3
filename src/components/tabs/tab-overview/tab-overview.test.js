import React from "react";
import renderer from "react-test-renderer";
import TabOverview from "./tab-overview.jsx";

const film = {
  genre: `action`,
  releaseDate: 1998,
  poster: ``,
  bgPoster: ``,
  ratingScore: `0,0`,
  ratingLevel: `OK`,
  ratingCount: 0,
  text: `text...`,
  director: `Steven Zaillian`,
  runTime: `1h 39m`,
  starring: `John Travolta, Robert Duvall, Stephen Fry`,
  reviews: []
};

it(`Should TabOverview renderer correctly`, () => {
  const tree = renderer
    .create(<TabOverview
      film = {film}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
