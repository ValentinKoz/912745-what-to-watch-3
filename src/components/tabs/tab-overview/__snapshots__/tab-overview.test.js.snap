// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TabOverview renderer correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      0,0
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        OK
      </span>
      <span
        className="movie-rating__count"
      >
        0 ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      text...
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: Steven Zaillian
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: John Travolta, Robert Duvall, Stephen Fry
      </strong>
    </p>
  </div>,
]
`;
